### ISSUE TITLE
[question] Multimodal input

### ISSUE THREAD

**Issue by Solliet** [hill-a/stable-baselines#784]
Hello,

What's the proper way to handle multi modal input? i.e. an image and a vector input? My current approach is to pretrain some other model (e.g. VAE) and wrap it around the environment, then concat the two. The downside of this is that the gradients aren't propagated.

**Comment by araffin** [comment]
Hello,
This may be a duplicate of #133 .
Short answer: you should use a custom policy, dict spaces are not supported yet.
Better support is planned for v3.1.

Using an autoencoder is a good solution, and I recommend that (see an [example](https://github.com/araffin/learning-to-drive-in-5-minutes))

**Comment by Solliet** [comment]
Hi,
I closed it as I found #133. I was mainly looking for ways to circumvent TF as my AE models are in PyTorch but I will figure it out. Thanks.

**Comment by araffin** [comment]
>I was mainly looking for ways to circumvent TF as my AE models are in PyTorch but I will figure it out.

Tensorflow will be soon not a pb anymore with v3 ;) (finally !)
but you can take a look at #372 .
